### PR TITLE
Devmode and related fixes

### PR DESCRIPTION
**Pull request by davidyuk**
This PR is supported by the Æternity Foundation

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -5,7 +5,6 @@ services:
       context: .
       dockerfile: ./Dockerfile
       args:
-        MIX_ENV: dev
```
Comment (davidyuk) [inline docker-compose-dev.yml:8]: For some reason with this line I have "Recv failure: Connection reset by peer" on 4000 port 🙁

File: Dockerfile [hunk 2]
```diff
@@ -152,4 +152,5 @@ USER ${USER}
 RUN rm -rf /home/aeternity/node/ae_mdw/log
 RUN mkdir -p /home/aeternity/node/ae_mdw/log
 
+HEALTHCHECK --start-period=10s --start-interval=2s --timeout=2s CMD /home/aeternity/healthcheck.sh
```
Comment (davidyuk) [inline Dockerfile:155]: related PR https://github.com/aeternity/aeternity/pull/4292

File: docker/aeternity-dev.yaml [hunk 3]
```diff
@@ -34,7 +34,7 @@ system:
   # plugin_path: /home/aeternity/node/plugins
   dev_mode: true
   plugins:
-    - name: ae_plugin_dev_mode
+    - name: aeplugin_dev_mode
```
Comment (davidyuk) [inline docker/aeternity-dev.yaml:37]: https://github.com/aeternity/aeplugin_dev_mode?tab=readme-ov-file#configuration
I'm not sure "ae_plugin_dev_mode" where comes from

File: docker-compose.yml [hunk 4]
```diff
@@ -1,11 +1,6 @@
 version: '3.1'
 services:
   ae_mdw:
-    healthcheck:
-      test: ["CMD", "/home/aeternity/healthcheck.sh"]
-      interval: 1m
-      timeout: 30s
-      retries: 3
```
Comment (sborrazas) [inline docker-compose.yml:8]: Why remove this? I think it's being used in production

**Review by yaboiishere — APPROVED** [review]

**Comment by sborrazas** [comment]
@davidyuk This PR LGTM, can you rebase for me to be able to merge it? 🙏

**Pull request merged**